src/main/java/seedu/address/model/delivery/Delivery.java

```diff
@@ -17,20 +17,21 @@ public class Delivery {
     private final Product product;
     private Person sender; // CHANGE TO SUPPLIER LATER ON
     private Status status;
-    private final Time deliveryTime;
+    private final DateTime deliveryTime;
     private final Cost cost;
     private final Quantity quantity;
     private final SupplierIndex supplierIndex;
 
     /**
      * Every field must be present and not null.
      */
-    public Delivery(Product product, Person sender, Status status, Time deliveryTime, Cost cost, Quantity quantity) {
-        requireAllNonNull(product, sender, status, deliveryTime, cost, quantity);
+    public Delivery(Product product, Person sender, Status status,
+                    DateTime deliveryDateTime, Cost cost, Quantity quantity) {
+        requireAllNonNull(product, sender, status, deliveryDateTime, cost, quantity);
         this.product = product;
         this.sender = sender;
         this.status = status;
-        this.deliveryTime = deliveryTime;
+        this.deliveryTime = deliveryDateTime;
```

Comment:
Should the instance field name be changed to deliveryDateTime as well for standardisation?